shell/dialog/RedeployWorkloadDialog.vue

```diff
@@ -0,0 +1,152 @@
+<script>
```

Comment:
I think that new components should be written in typescript when possible.

Reply:
Sure, updated in commit b690b8ce7cd552847d551374f30ac0de58369f80.